homepage/migrations/0006_review_test_data.py

```diff
@@ -15,30 +18,39 @@ def generate_data(apps, schema_editor):
         from homepage.models import Course
         from homepage.models import Review
         from homepage.models import Professor
-        from django.utils import timezone
         from django.core.files.uploadedfile import SimpleUploadedFile
+        from os import path
         from django.conf import settings
+        from datetime import datetime
+        import pytz
 
         users = AppUser.objects.all()
         courses = Course.objects.all()
         professors = Professor.objects.all()
+        image = ''
 
-        image_path = settings.MEDIA_ROOT + '/images/test_image.jpg'
-        image = SimpleUploadedFile(
-            name='new_test_image.jpg',
-            content=open(image_path, 'rb').read(),
-            content_type='image/jpeg')
+        # prevent creating a new image upon every pytest run
+        if not path.exists(path.join(settings.MEDIA_ROOT, NEW_IMAGE_FILE)):
+            image_path = path.join(settings.MEDIA_ROOT, IMAGES_DIR, IMAGE_FILE)
+            image = SimpleUploadedFile(
+                name=NEW_IMAGE_FILE,
+                content=open(image_path, 'rb').read(),
+                content_type='image/jpeg')
+        else:
+            image = NEW_IMAGE_FILE
+
+        date = datetime(2015, 10, 9, 23, 55, 59, 5, tzinfo=pytz.UTC)
```

Comment:
the "problem" in python is that you can overwrite basic python keywords easily by assign them to something else
for example:
In [1]: str(2)                                                                                                                                                                                                                               
Out[1]: '2'

In [2]: str = 'string'                                                                                                                                                                                                                       

In [3]: str(2)                                                                                                                                                                                                                               
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-3-531f7a6f46fa> in <module>
----> 1 str(2)

TypeError: 'str' object is not callable

here is not the case but because you imported datetime and it has the "date" method I would suggest to change the name to something less generic 

```suggestion
        cur_date = datetime(2015, 10, 9, 23, 55, 59, 5, tzinfo=pytz.UTC)
```

Reply:
Thank you for this significant insight. 
I'll be aware of that in the future. 
Changing the variable name.